packages/cli/src/oclif/commands/test.js

```diff
@@ -57,29 +73,27 @@ class TestCommand extends BaseCommand {
 
 TestCommand.flags = buildFlags({
   commandFlags: {
-    timeout: flags.integer({
-      char: 't',
-      description: 'Set test-case timeout in milliseconds.',
-      default: 2000,
-    }),
-    grep: flags.string({
-      description: 'Only run tests matching pattern.',
-    }),
     'skip-validate': flags.boolean({
-      description: 'Forgo running `zapier validate` before `npm test`.',
+      description:
+        "Forgo running `zapier validate` before tests are run. This will speed up tests if you're modifying functionality of an existing integration rather than adding new actions.",
     }),
     yarn: flags.boolean({
-      description: 'Use yarn instead of npm.',
+      description:
+        "Use `yarn` instead of `npm`. This happens automatically if there's a `yarn.lock` file, but you can manually force `yarn` if you run tests from a sub-directory.",
     }),
   },
 });
 
+TestCommand.strict = false;
```

Comment:
unfortunately, this is required to allow arbitrary extra args. Downside is that if a users typos `skip-validate`, it'll be silently ignored. But, I think there's enough upside here. The FR has been filed in the parser as https://github.com/oclif/parser/issues/74